pkg/grpc/client.go

```diff
@@ -13,9 +15,10 @@ type ClientConnection struct {
 	conn   *grpc.ClientConn
 }
 
-func ConnectClient(address string) (*ClientConnection, error) {
+func ConnectClient(address string, port int) (*ClientConnection, error) {
 	// TODO: allow configuring some options (keepalive, backoff...)
-	conn, err := grpc.Dial(address,
+	grpcSocket := fmt.Sprintf("[%s]:%d", address, port)
```

Comment:
Square brackets should only be added if g.hostIP is an IPv6 address, otherwise it should just be "%s:%d".

Reply:
I thought [] should work for both AF ? anyway I will make it AF specific